double_pendulum/double_pendulum_approximation.py

```diff
@@ -1,14 +1,17 @@
 import pyro
 import torch
 from torch import nn
-from torchdiffeq import odeint_adjoint as odeint
+from torchdiffeq import odeint
```

Comment:
Potentially slowdown in training and increased memory usage. If second derivatives are not needed better to use  odeint_adjoint

Reply:
OK